hc/front/tests/test_add_channel.py

```diff
@@ -36,6 +36,13 @@ def test_instructions_work(self):
             url = "/integrations/add_%s/" % frag
             r = self.client.get(url)
             self.assertContains(r, "Integration Settings", status_code=200)
+            
+    def test_bad_kinds_dont_work(self):
```

Comment:
The doc string doesn't really explain much about what the test does, think this should have contained more info